README.md

```diff
@@ -33,21 +33,21 @@ This Repository is for the NHLBI BDC Metrics Dashboard. A proof of Concept of th
 ### Logins:
 * Dug: N/A
 * PIC-SURE: Total logins to Authorized PIC-SURE via eRA Commons or BDC Developer Login
-* Gen3: Total Logins to Gen3, via NIH login or BDC Developer Login
+* Gen3: Total Logins to the BDC Ecosystem, via NIH login or BDC Developer Login
 * Terra: Total Logins to Terra via Google or Microsoft
 * Seven Bridges: Total logins to Seven Bridges via eRA Commons or BDC Developer Login
 
 ### Visits:
 * Dug: Total visits to [Dug](https://search.biodatacatalyst.renci.org/)
 * PIC-SURE: Total Visits to [Open PIC-SURE](https://openpicsure.biodatacatalyst.nhlbi.nih.gov/picsureui/)
-* Gen3: N/A
+* Gen3: Total unique user visits to [Gen3 Data Portal](https://gen3.biodatacatalyst.nhlbi.nih.gov/)
 * Terra: N/A
 * Seven Bridges: N/A
 
 ### Search Events:
 * Dug: Typing a text string in the search bar and hitting Search
 * PIC-SURE: The total number of searches (A text string typed into the search bar and searched) and queries (Using the database filter tools to find a specific set of data)
-* Gen3: The number of Explorer Filters used on Gen3’s Exploration page.
+* Gen3: N/A
```

Comment:
Gen3 does not currently have an easy way to filter out search events that were from a user vs. a search event that was executed to populate the data portal UI.